islandora.module

```diff
@@ -2170,3 +2170,23 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $object = islandora_object_load($cmodel);
+      $rows[] = array(l($object->label, "islandora/object/{$object->id}"));
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
```

Comment:
This should probably use [`format_plural()`](https://api.drupal.org/api/drupal/includes!common.inc/function/format_plural/7.x). Would look something like:
```php
$headers = format_plural(count($rows), 'Content Model', 'Content Models');
```